### PR TITLE
Fix SyntaxWarning over comparison of literals using is

### LINKED ISSUE #66: Syntax warning over comparison of literals using is
```
find . -iname '*.py'  | xargs -P 4 -I{} python -Wall -m py_compile {}

./chevron/renderer.py:78: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if scope is 0:  # noqa: F632
```

### PR DESCRIPTION
**Pull request by tirkarthi**
Fixes #66

### CODE REVIEW

**Comment by smurfix** [comment]
Is there a problem with this merge request?

This triggers a SyntaxWarning on Python 3.8. Please release a fixed version ASAP.

File: chevron/renderer.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def _get_key(key, scopes):
             # Return an empty string if falsy, with two exceptions
             # 0 should return 0, and False should return False
             # While using is for this check is undefined it works and is fast
-            if scope is 0:  # noqa: F632
+            if scope == 0:  # noqa: F632
```
Comment (smurfix) [inline chevron/renderer.py:78]: Please remove the noqa: comment, it no longer applies.

Comment (smurfix) [reply chevron/renderer.py:78]: (in fact the problem should have been fixed, instead of adding that comment …)

Comment (tirkarthi) [reply chevron/renderer.py]: Thanks, fixed.

File: chevron/renderer.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def _get_key(key, scopes):
             # Return an empty string if falsy, with two exceptions
             # 0 should return 0, and False should return False
             # While using is for this check is undefined it works and is fast
```
Comment (dcmoore-gd) [inline chevron/renderer.py:77]: Should probably remove this comment now that `is` isn't being used anymore?

Comment (tirkarthi) [reply chevron/renderer.py]: Fixed now. Thanks.

Comment (dcmoore-gd) [reply chevron/renderer.py]: Thank you @tirkarthi for doing this PR!

**Comment by mrichman** [comment]
I still get this warning using 0.13.1: `/var/task/chevron/renderer.py:73: SyntaxWarning: "is" with a literal. Did you mean "=="?`

**Comment by tim-pugh** [comment]
@mrichman I don't believe it was pulled into the master branch yet, but only made it's way through the CI/CD pipeline (maybe I'm wrong here assuming that). However, from what I can see:

https://github.com/noahmorrison/chevron/blob/78f1a384eddef16906732d8db66deea6d37049b7/chevron/renderer.py#L78

it doesn't appear so. I also see you raised issue #75; I'll try to jump on that one and ask the other dev to peek in here and see if my assumption is correct and see if they have the correct permissions to merge, unless someone else is reading this <:o)

**Comment by theshadow27** [comment]
+1 on the issue with 3.8. Is there an ETA on getting this merged?

**Comment by noahmorrison** [comment]
[merged](https://github.com/noahmorrison/chevron/commit/d43fabb2ebfa8c79791ac5b1f74bf00fd07cb57b)

**Pull request closed without merging**